### PR TITLE
Freeipa

### PR DESCRIPTION
**Pull request by janzagata**
Freeipa service added

### CODE REVIEW

File: slave-new/process-freeipa/bin/process-freeipa.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+PROTOCOL_VERSION='3.0.0'
+
+
+function log_both {
```
Comment (stavamichal) [inline slave-new/process-freeipa/bin/process-freeipa.sh]: We don't need this function. Can be removed.

File: slave/process-freeipa.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+function process {
+
+	FROM_PERUN="${WORK_DIR}/freeipa"
+	EXEC_SCRIPT="${SCRIPTS_DIR}/freeipa/process-freeipa.py"
+
+	create_lock
+	
+	python2 $EXEC_SCRIPT --perun-file $FROM_PERUN --user $USER --password $PASSWORD --host-url $IPA_HOST
+
```
Comment (stavamichal) [inline slave/process-freeipa.sh]: There need to be "exit $?".

File: slave/process-freeipa.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+PROTOCOL_VERSION='3.0.0'
+
+
+function log_both {
```
Comment (stavamichal) [inline slave/process-freeipa.sh]: We don't need this function. Can be removed.

**Comment by stavamichal** [comment]
There are also some conflicts in this branch. Please resolve them (probably something with missing "voms" in rule files.

File: slave/freeipa/freeipa_slave.py [hunk 4]
```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
```
Comment (stavamichal) [inline slave/freeipa/freeipa_slave.py]: please rename this script to process-freeipa.py

File: slave-new/process-freeipa/lib/freeipa_slave.py [hunk 5]
```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
```
Comment (stavamichal) [inline slave-new/process-freeipa/lib/freeipa_slave.py]: please rename this script to process-freeipa.py

**Comment by licehammer** [comment]
@janzagata We need to resolve comments in this pull request in order to merge it into our repo and create installation package for you. Alternatively, we can reject this pull request and you can manage this script manually (same way as it is now). What do you prefer?

**Comment by stavamichal** [comment]
Already merged in other pull request.
I am closing it.

**Pull request closed without merging**